docs/DeveloperGuide.md

```diff
@@ -1,29 +1,134 @@
 # Developer Guide
 
+## Welcome!
+Welcome and thanks for joining us in building our program created to handle all of your currencies and money problems. With us you can use such functions as transfer, withdraw, add with different currencies to handle all your daily money problems. You also have the opportunity to add your own currencies, such as cryptocurrencies and etc. 
+
+Within this documentation you can find all the information needed to keep building it.
+
+## Setting up
```

Comment:
Some links to the different parts of the DG would perhaps be useful for navigation.